SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Source.cs

```diff
@@ -222,7 +298,121 @@ public static void ForceCurrent(this DCPowerSessionsBundle sessionsBundle, doubl
         }
 
         /// <summary>
-        /// Forces current and specifies symmetric voltage limits.
+        /// Forces current on the target pins at the specified level. Must at least provide a level value, and the method will assume all other properties that have been previously set. Optionally, can also provide a specific voltage limit, current level range, voltage limit range values directly.
```

Comment:
Suggest changing to: "Forces current on the target pins at the specified pin-unique level."